pycaption/scc/constants.py

```diff
@@ -983,5 +983,7 @@ def _restructure_bytes_to_position_map(byte_to_pos_map):
 # Time to transmit a single codeword = 1 second / 29.97
 MICROSECONDS_PER_CODEWORD = 1000.0 * 1000.0 / (30.0 * 1000.0 / 1001.0)
 
+def MICROSECONDS_PER_CODEWORD_fps(src_fps=30.0):
```

Comment:
This is a function, not a constant, so it shouldn't be declared with capital letters. Please use only lowercase for function names as per [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)